### ISSUE TITLE
Incorrect casing in filter method name conversion.

### ISSUE THREAD

**Issue by canerelci** [dotliquid/dotliquid#136]
In Turkish language, and maybe in different languages, some common latin letters have different lower letters. In my example, lowercase of letter 'I' is not 'i', but 'ı'. This makes some filters inaccessible while using ruby casing conventions. Here is the fix:

In RubyNamingConvention.cs: 

```c#
@@ -26,7 +26,7 @@ public StringComparer StringComparer
		public string GetMemberName(string name)
		{
			// Replace any capital letters, apart from the first character, with _x, the same way Ruby does
-			return _regex2.Replace(_regex1.Replace(name, "$1_$2"), "$1_$2").ToLower();
+			return _regex2.Replace(_regex1.Replace(name, "$1_$2"), "$1_$2").ToLowerInvariant();
		}
	}
}
```

**Comment by Elph** [comment]
Yep, I had the same problem. That was fixed on #170

**Comment by tgjones** [comment]
Thanks, closing this as it's fixed in #170.
